### DISCUSSION TITLE
After installing BlackHole, system alerts are replaced by screen flash

### DISCUSSION THREAD

**Discussion by renauddetry** [ExistentialAudio/BlackHole#788] [General]
Many thanks for BlackHole, it's great!

One tiny thing: after installing, I no longer get a bell sound for system alerts. Instead, the screen flashes once.

Notes:
- the system pref option _Flash the screen when an alert sound occurs_ is set to off.
- In Audio Midi Setup, MacBook Pro Speakers is set to play alerts and sound effects.

If I uninstall BlackHole with the instruction provided in REAME, audio alerts resume normally.

**Comment by devinroth** [comment]
BlackHole doesn’t have access to alter any of those settings. Do you have ACE driver installed? I recently noticed some messed up stuff with Rogue Amoeba due to their kernel extension.

Devin

**Comment by gchilds** [comment]
RA have gone driver free, so either ScreenCaptureKit (unlikely) or CATap.

**Comment by devinroth** [comment]
I wish they would go kext free.

Devin

**Comment by gchilds** [comment]
https://weblog.rogueamoeba.com/2024/04/05/our-new-installer-free-setup-comes-to-audio-hijack/
